framework_tvb/tvb/core/services/project_service.py

```diff
@@ -156,6 +156,12 @@ def find_project(self, project_id):
             self.logger.exception("Given Project ID was not found in DB!")
             raise ProjectServiceException(str(excep))
 
+    def find_project_path(self, project_name):
```

Comment:
I think this new method is never used. I will remove it.

Reply:
I forgot it there...